src/lazy/text/buffer.rs

```diff
@@ -1337,6 +1343,28 @@ impl<'data> TextBufferView<'data> {
             recognize(pair(Self::match_any_digit, Self::match_any_digit)),
         )(self)
     }
+
+    /// Matches a complete blob, including the opening `{{` and closing `}}`.
+    pub fn match_blob(self) -> IonParseResult<'data, MatchedBlob> {
+        delimited(
+            tag("{{"),
+            // Only whitespace (not comments) can appear within the blob
+            preceded(Self::match_optional_whitespace, Self::match_base64_content),
+            preceded(Self::match_optional_whitespace, tag("}}")),
```

Comment:
> Within blob values, whitespace is ignored.

That means something like `{{ cm F6emx lIGRhenps  ZSByb29  0IGJlZ XI= }}` should be matched successfully, right? 

Also, it looks like the `match_base64_content` function does not match an empty input/string, and the base64 content is not optional here, so I don't think this would match `{{}}` as it should.